TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -4000,7 +4012,9 @@ private struct BufferItem {
                 laneInfo,
                 VehicleRestrictionsMode.Configured);
 
-            return (allowedTypes & queueItem_.vehicleType) != ExtVehicleType.None;
+            return (allowedTypes & queueItem_.vehicleType) != ExtVehicleType.None ||
+                   startLaneA_ == laneId || startLaneB_ == laneId ||
+                   endLaneA_ == laneId || endLaneB_ == laneId;
```

Comment:
`vehicleRestrictionsManager.GetAllowedVehicleTypes()` is more expensive than the start/end lane checks - as such should the start/end lane checks be moved to the end of the first `if` block?

Reply:
Hmm, good idea!

Reply:
Actually... It won't make a big difference -> it will be `true` only for first and last path buffer item since it starts from known position and ends immediately when end position is found (which sometimes may cause issues 😕 )

> Ok, so no need to update pathfinder edition number or despawn any vehicles when user loads savegame from prior pathfinder.

Yes, no need to clear anything.